selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/gatein/GateinLocator.java

```diff
@@ -1274,8 +1276,8 @@
   public static final String          ELEMENT_SEARCH_ICON_USERS_MANAGEMENT                    =
                                                                            "//*[contains(@title,'Quick Search')]";
 
-  public static final String          ELEMENT_CLOSE_MESSAGE                                   =
-                                                            "//*[contains(@title,'Close Window')]";
+  public static final SelenideElement       ELEMENT_CLOSE_MESSAGE                                   =
+          $(byText("Close"));
```

Comment:
Don't modify an existed locator, it may impact other tests. if you can't reuse it, add a new one.